apps/docs/src/app/core/component-docs/object-marker/examples/object-marker-Icon-text-example.component.html

```diff
@@ -0,0 +1,5 @@
+<span fd-object-marker [glyph]="'add-favorite'">Favourite</span>
```

Comment:
`[glyph]="'add-favorite'"` can be `glyph="add-favorite"` right? I think for string values we are not keeping the binding.